src/lib/ObjectDefinitions/Comment.js

```diff
@@ -26,3 +28,8 @@ Comment.prototype.createdAtTime = function(createdAt){
     this.createdAt = createdAt;
     return this;
 };
+
+Comment.prototype.withProfile = function(image) {
```

Comment:
rename to withImage, withAvatar, withProfileImage, hasProfileImage, etc...
withProfile is kind of a confusing name in this context